### ISSUE TITLE
Noto font fallback suggestions

### ISSUE THREAD

**Issue by imnasnainaec** [silnrsi/fonts#1]
There are 3 Google Noto fonts in https://github.com/silnrsi/fonts/blob/main/families.json that are no longer publicly available but appear to have good fallback options:

- ["Noto Sans Meroitic Cursive"](https://github.com/silnrsi/fonts/blob/main/families.json#L4758-L4765) -> ["Noto Sans Meroitic"](https://github.com/silnrsi/fonts/blob/main/families.json#L4740-L4757), which is listed as the Default Font for the "Meroitic Cursive" script in https://github.com/silnrsi/langfontfinder/blob/main/data/script2font.csv#L113
- ["Noto Sans Tangut"](https://github.com/silnrsi/fonts/blob/main/families.json#L5872-L5879) -> ["Noto Serif Tangut"](https://fonts.google.com/noto/specimen/Noto+Serif+Tangut), which would need to be added to the file
- ["Noto Sans Tibetan"](https://github.com/silnrsi/fonts/blob/main/families.json#L6056-L6063) -> ["Noto Serif Tibetan"](https://github.com/silnrsi/fonts/blob/main/families.json#L7072-L7097)

**Comment by jvgaultney** [comment]
Thanks - there were issues a while back about the Noto fonts for these scripts, so we removed the some of the data until things could settle. It's now time to fix up records for these in families.json. Here's what needs doing (listed by proposed or current familyid):

_Note that these changes alone would not affect script-to-font associations in the Language Font Finder service. That is a separate process in a different repo._

_notosansmeroitic_: Base file exists and no changes needed - it is already correct in base file and families.json

_notosansmeroiticcursive_: Base file needs "status" changed to "deprecated" and added  "fallback": "notosansmeroitic"

_notoseriftangut_: Needs a new base file with data based on Google's state.json file.

_notosanstangut_: Base file needs "status" changed to "deprecated" and added  "fallback": "notoseriftangut"

_notoseriftibetan_: Base file exists and no changes needed - it is already correct in base file and families.json
 
_notosanstibetan_: Base file needs "status" changed to "deprecated" and added  "fallback": "notoseriftibetan"

**Comment by imnasnainaec** [comment]
@jvgaultney Cool! Would if be helpful for me to submit a pr with those changes?

**Comment by jvgaultney** [comment]
Already done. We were due to update the info on all the Noto families we're tracking so I did these at the same time.

In the future you would be very welcome to submit pull requests. The main thing to be careful about is that you edit only the basefiles, then run the updatefamilydata.py script to update families.json.  See the [maintenance.md](https://github.com/silnrsi/fonts/blob/main/documentation/maintenance.md) documentation.
